src/api/CourseManagement.ts

```diff
@@ -29,6 +34,9 @@ export default class CourseManagement extends Common {
         if (moduleIds != undefined) {
             requestParameter.params.moduleIds = moduleIds.reduce((a, b) => a + "," + b, "");
         }
+        if (examregNames != undefined) {
+            requestParameter.params.examregNames = examregNames.reduce((a, b) => a + "," + b, "");
```

Comment:
Do we need the same fix here? (removing the last' ""')

Reply:
No, this is a lagom bug and just "unexpected" use of the API, but still perfectly fine in the API definition